packages/webdriverio/README.md

```diff
@@ -1,5 +1,140 @@
 # @axe-core/webdriverio
 
-Provides bindings for axe with webdriverio
+> Provides a chainable axe API for WebdriverIO and automatically injects into all frames.
 
-Previous versions of this program were maintained at [dequelabs/axe-webdriverio](https://github.com/dequelabs/axe-webdriverio).
+## Getting Started
+
+Install [Node.js](https://docs.npmjs.com/getting-started/installing-node) if you haven't already.
+
+> Download and install any necessary browser drivers on your machine's PATH. [More on WebdriverIO setup](https://v6.webdriver.io/docs/gettingstarted.html#taking-the-first-step).
+
+Install `@axe-core/webdriverio` and its dependencies:
+
+NPM:
+
+```console
+npm install @axe-core/webdriverio
+```
+
+Yarn:
+
+```console
+yarn add @axe-core/webdriverio
+```
+
+## Usage
+
+This module uses a chainable API to assist in injecting, configuring, and analyzing axe with WebdriverIO. As such, it is required to pass an instance of WebdriverIO.
+
+Here is an example of a script that will drive WebdriverIO to a page, perform an analysis, and then log results to the console.
+
+```js
+const AxeBuilder = require('@axe-core/webdriverio').default;
+const { remote } = require('webdriverio');
+
+(async () => {
+  const client = await remote({
+    capabilities: {
+      browserName: 'firefox'
+    }
+  });
+
+  await client.url('https://dequeuniversity.com/demo/mars/');
+
+  const builder = new AxeBuilder({ client });
+  try {
+    const results = await builder.analyze();
+    console.log(results);
+  } catch (e) {
+    console.error(e);
+  }
+})();
+```
+
+## AxeBuilder({ client: WebdriverIO.BrowserObject })
```

Comment:
Nitpick, all of these API methods should be nested under a level 2 heading API and the rest should be level 3 in my opinion.